tracks/python/exercises/armstrong-numbers/mentoring.md

```diff
@@ -37,8 +37,8 @@ def is_armstrong_number(number):
     # With a generator expression passed to sum
     total = sum(digit ** count for digit in digits)
 ```
-Students are often unaware of [generator expressions][pep-289] and that they can be passed directly to functions that accept iterables.
-The first line uses a comprehension to build a `list`  for `sum()` to iterate through.
+Students are often unaware of [generator expressions][pep-289] and that they can be passed directly to functions that accept iteration.
```

Comment:
Is this intentionally part of this PR?

Reply:
Oh sorry about that. I committed to the main branch. it was a mistake.